tests/jobs/test_scheduler_job.py

```diff
@@ -3434,6 +3435,12 @@ def test_timeout_triggers(self, dag_maker):
         assert ti1.next_method == "__fail__"
         assert ti2.state == State.DEFERRED
 
+    @pytest.fixture(autouse=True, scope='function')
+    def clean_listener(self):
+        get_listener_manager().clear()
+        yield
+        get_listener_manager().clear()
```

Comment:
Why is this needed in SchedulerJob tests?

Reply:
Removed. Artifact from debugging...